src/main/java/io/github/intoto/models/Predicate.java

```diff
@@ -0,0 +1,18 @@
+package io.github.intoto.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+/**
+ * A generic attestation type with a schema isomorphic to in-toto 0.9. This allows existing in-toto
```

Comment:
Is this comment right here?  I'm not sure it applies (and if it does I'm confused).

Reply:
Done.